openlibrary/data/sitemap.py

```diff
@@ -18,6 +18,7 @@
 import datetime
 from gzip import open as gzopen
 
+import simplejson
```

Comment:
we may want to check json v. simplejson

Reply:
Can we open a _separate_ PR on this one.  One word of caution is that I have seen _json_ used as a variable name which will trip us up if we do a global find and replace of __simplejson —> json__.

Reply:
#1451

Reply:
#1451